apps/game_backend/priv/curse_of_mirra/quests_descriptions.json

```diff
@@ -88,8 +89,9 @@
     },
     {
         "config_id": 5,
-        "description": "Skill Issue",
+        "description": "Deal 300 damage",
         "type": "bounty",
+        "quest_type": "damage",
```

Comment:
I would put this type inside objective.
So we move from `Quest.quest_type` (Quest.type exists too) to `Quest.objective.type`.
What do you think?